### ISSUE TITLE
Identify gaps place holders and photos etc

### ISSUE THREAD

**Issue by ArryoRuma** [TruMedia-Creative/tmc#121]
I need you to go thru and create a md file that goes thru at each level based off type of thing that is missing like photos and where that file is located/needs to be updated. Or copy or placeholder text or alt text etc.
